.github/workflows/ci-unit-tests.yml

```diff
@@ -45,3 +61,13 @@ jobs:
         flags: unittests
         fail_ci_if_error: true
         token: ${{ env.CODECOV_TOKEN }}
+
+  event_file:
```

Comment:
what does this do?

Reply:
Oops, good catch, that was me experimenting with https://github.com/EnricoMi/publish-unit-test-result-action/blob/v2.12.0/README.md#support-fork-repositories-and-dependabot-branches and was an accidental half-baked change. Reverted in [396fb72](https://github.com/jaegertracing/jaeger/pull/5030/commits/396fb72f58296704bf35dbc6dbfbbe44f19048dd) and [aa357c1](https://github.com/jaegertracing/jaeger/pull/5030/commits/aa357c1c97b9925e40d0d4193d08d7e38a46e6c2).

That may have been why we could see the test report in https://github.com/jaegertracing/jaeger/actions/runs/7315855796?pr=5030#summary-19929936699, but I'll check once the test CI job is done.